src/org/zaproxy/zap/extension/callback/ExtensionCallback.java

```diff
@@ -94,6 +95,7 @@ public void hook(ExtensionHook extensionHook) {
                     getCallbackPanel());
             extensionHook.getHookView().addOptionPanel(
                     getOptionsCallbackPanel());
+            ExtensionHelp.enableHelpKey(getOptionsCallbackPanel(), "ui.tabs.callbacks");
```

Comment:
Actually should be `getCallbackPanel()` instead?

Reply:
That’s what I get for rushing, tweak coming